tests/test_computed_fields.py

```diff
@@ -795,3 +795,16 @@ def id(self) -> str:
 
     m = Model(bar=42)
     assert m.model_dump() == {'bar': 42, 'id': 'id: {"bar":42}'}
+
+
+def test_computed_field_dependencies():
+    class Model(BaseModel):
+        width: float
+        height: float
+
+        @computed_field(dependencies=['width', 'height'])
```

Comment:
How should nested fields be specified? e.g.:

```python
class Sub(BaseModel):
    foo: int

class Model(BaseModel):
    sub: Sub

    @computed_field(dependencies=[???])
    @property
    def prop(self) -> int:
        return self.sub.int
```

Reply:
ah, very good question. 
i guess the two possibilities that come to mind are:

- follow the same `loc` syntax that pydantic uses when expressing Validation errors, with dotted attributes  `dependencies=['sub.foo']`
- allow deps to be a dict as well `dependencies={'sub': ['foo']}`
- or... just say "you can't": *all* names in dependencies must be field names on this model `dependencies=['sub']`

I admit that None of those make me jump for joy, and that does definitely raise an issue for the whole pattern.  What do you think?

Reply:
I'd be inclined to use a pattern similar to that of `exclude` as a serialization flag:

https://docs.pydantic.dev/latest/concepts/serialization/#advanced-include-and-exclude